ref/general/configuring-spnego-auth.adoc

```diff
@@ -186,7 +194,7 @@ The runtime forms the default SPN in the following format:
 "HTTP/" + java.net.InetAddress.getLocalHost().getCanonicalHostName();Copy
 ----
 +
-If the default SPN does not match what you have in the krb5.keytab file, then you need to specify the servicePrincipalNames, for example:
+If the default SPN does not match what you have in the `krb5.keytab` file, then you need to specify the `servicePrincipalNames`, for example:
```

Comment:
what kind of element is `servicePrincipalNames`? an attirbute? needs an identifier